api/backend/ecs/load_balancer_manager.go

```diff
@@ -220,6 +220,7 @@ func (e *ECSLoadBalancerManager) listenerToPort(listener *awselb.Listener) model
 	}
 
 	if listener.SSLCertificateId != nil {
+		port.CertificateARN = *listener.SSLCertificateId
```

Comment:
Are we sure this value will be the ACM certificates ARN and not it's ID? Also, don't we need some sort of update to terraform to ensure these values sync up (or is purposely omitted?)

Reply:
According to the documentation, yes. https://docs.aws.amazon.com/sdk-for-go/api/service/elb/#Listener

> The Amazon Resource Name (ARN) of the server certificate.

L0 Provider doesn't require any changes.